python/GafferSceneTest/PassWedgeTest.py

```diff
@@ -0,0 +1,242 @@
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import inspect
+import unittest
+
+import IECore
+
+import Gaffer
+import GafferTest
+import GafferDispatch
+import GafferDispatchTest
+import GafferScene
+
+class PassWedgeTest( GafferTest.TestCase ) :
+
+	def __dispatcher( self, frameRange = None ) :
+
+		result = GafferDispatch.LocalDispatcher()
+		result["jobsDirectory"].setValue( self.temporaryDirectory() / "jobs" )
+
+		if frameRange is not None :
+			result["framesMode"].setValue( result.FramesMode.CustomRange )
+			result["frameRange"].setValue( frameRange )
+
+		return result
+
+	def testPassNames( self ) :
+
+		script = Gaffer.ScriptNode()
+
+		script["writer"] = GafferDispatchTest.TextWriter()
+		script["writer"]["fileName"].setValue( self.temporaryDirectory() / "${pass}.txt" )
+
+		script["wedge"] = GafferScene.PassWedge()
+		script["wedge"]["preTasks"][0].setInput( script["writer"]["task"] )
+
+		script["passes"] = GafferScene.Passes()
+		script["passes"]["names"].setValue( IECore.StringVectorData( [ "tom", "dick", "harry" ] ) )
+
+		script["wedge"]["in"].setInput( script["passes"]["out"] )
+
+		self.__dispatcher().dispatch( [ script["wedge"] ] )
+
+		self.assertEqual(
+			set( self.temporaryDirectory().glob( "*.txt" ) ),
+			{
+				self.temporaryDirectory() / "tom.txt",
+				self.temporaryDirectory() / "dick.txt",
+				self.temporaryDirectory() / "harry.txt",
+			}
+		)
+
+	def testSkipDisabledPasses( self ) :
+
+		script = Gaffer.ScriptNode()
+
+		script["writer"] = GafferDispatchTest.TextWriter()
+		script["writer"]["fileName"].setValue( self.temporaryDirectory() / "${pass}.txt" )
+
+		script["wedge"] = GafferScene.PassWedge()
+		script["wedge"]["preTasks"][0].setInput( script["writer"]["task"] )
+
+		script["passes"] = GafferScene.Passes()
+		script["passes"]["names"].setValue( IECore.StringVectorData( [ "tom", "dick", "harry" ] ) )
+
+		script["disablePass"] = GafferScene.CustomOptions()
+		script["disablePass"]["in"].setInput( script["passes"]["out"] )
+		script["disablePass"]["options"].addChild( Gaffer.NameValuePlug( "", Gaffer.BoolPlug( "value", defaultValue = False ), True, "member1" ) )
+		script["disablePass"]["options"]["member1"]["name"].setValue( "pass:enabled" )
```

Comment:
I think this line can be omitted if we replace "" with "pass:enabled" on the line above?

Reply:
Simplified in 4572e7c.